manifests/init.pp

```diff
@@ -142,18 +152,23 @@
   @service { 'ipa':
     ensure  => 'running',
     enable  => true,
-    require => Package[$ipa::svrpkg]
+    require => Package[$pkglist],
```

Comment:
According to the style guide (https://docs.puppet.com/guides/style_guide.html#variables) local variables don't need to be fully qualified.

I'll take a look at the other comments in the morning. Thanks for the feedback!